examples/histogram.md

```diff
@@ -0,0 +1,114 @@
+## Logging Histogram
+
+These examples help you to understand `SummaryWriter::Histogram()` API in depth.
+
+### 0. API 
+
+  1. [Log Histogram](#1-histogram)
+  2. [Log Histogram stored in arma::vec](#2-histogram-arma-vec)
+
+### 1. Histogram
+
+A histogram could be logged using the following API:
+
+```cpp
+void Histogram(const std::string& tag,
+               int step,
+               std::vector<double>& values,
+               Filewriter& fw);
+```
+The API accepts `tag`, `step`, `values` and `filewriter` object.
+
+Following is a snippet that would log some histogram values.
+
+```cpp
+#include <mlboard/mlboard.hpp>
+#include <iostream>
+#include <chrono> 
+#include <ctime> 
+#include <future>
+#include <random>
+
+int main()
+{
+  // Creating a FileWriter object that is responsible for logging the summary.
```

Comment:
The histogram which is getting logged by running this example is different that which is shown in the below image.